src/main/java/io/stargate/sgv2/jsonapi/service/processor/MeteredCommandProcessor.java

```diff
@@ -138,22 +131,19 @@ public <T extends Command> Uni<CommandResult> processCommand(
    * @param result Command result
    * @return Command log in string format
    */
-  private String buildCommandLog(
-      CommandContext commandContext, Command command, CommandResult result) {
+  private <T extends SchemaObject> String buildCommandLog(
+      CommandContext<T> commandContext, Command command, CommandResult result) {
     CommandLog commandLog =
         new CommandLog(
             command.getClass().getSimpleName(),
             dataApiRequestInfo.getTenantId().orElse(UNKNOWN_VALUE),
-            commandContext.namespace(),
-            commandContext.collection(),
+            commandContext.schemaObject().name.keyspace(),
+            commandContext.schemaObject().name.table(),
+            commandContext.schemaObject().type.name(),
             getIncomingDocumentsCount(command),
             getOutgoingDocumentsCount(result),
             result != null ? result.errors() : Collections.emptyList());
-    try {
-      return OBJECT_WRITER.writeValueAsString(commandLog);
-    } catch (JacksonException e) {
-      return "ERROR: Failed to serialize CommandLog instance, cause = " + e;
-    }
+    return new ObjectMapper().valueToTree(commandLog).toString();
```

Comment:
Actually I think this is true reversal, as chunk 4 still has the change!

We need to figure out how to fix the problem; I saw 2 accidental reversals but there could be more. Unfortunately these 2 not covered by unit tests (one is performance fix; the other for stack trace observed in prod env, not during testing), otherwise would have been caught.

Reply:
Change reverted was https://github.com/stargate/data-api/pull/1269

Reply:
Will verify in chunk/4 that both changes retained.